### ISSUE TITLE
Power Management, Wallpaper, and Mute issues.

### ISSUE THREAD

**Issue by MasterTextman** [linuxmint/Cinnamon#7116]
```
 * Cinnamon version (cinnamon --version) : 3.2.7
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) : Ubuntu MATE 17.04 Zesty Zapus
 * Graphics hardware *and* driver used : Intel HD Graphics 3000, i915
 * 32 or 64 bit : 64 bit.
 ```

**Issue**

Multiple issues actually.
**1. Power Management won't work,**
I cannot launch it from the settings menu, nor can I actually make it show up on the panel. Disabling and re-enabling multiple times doesn't work.

**2. Background is just black.**
I can't change the background at all. I tried everything, from within the settings menu, to pressing "set as desktop background", to even using pywal. Nothing works.

**3. Mute is completely broken.**
I tried setting a custom shortcut in the keyboard settings. Lowering volume, making volume higher, etc all work perfectly. However, volume mute does not work at all. I tried taking the binding off the other (non-working) mute option, or even just leaving it without an assigned key, and my custom shortcut still won't work. The command is `pactl set-sink-mute 0 toggle`. Putting it in the terminal works perfectly.

**Steps to reproduce**
All I did was install the latest Cinnamon version from the repository embrosyn/cinnamon.

**Expected behaviour**
Well, I expect to be able to mute, change the background, and have the actual power management settings work.

**Other information**
I AM using Cinnamon on a T420, but I don't think that should cause much issues.

**Comment by collinss** [comment]
You should open a separate issue for each bug. This sort of collect-all bug report will probably be closed immediately. Also, you didn't really include any useful information at all. You might actually consider using the issue template if you want your bug reports to be taken seriously.

**Comment by MasterTextman** [comment]
But I did use the issue template. What are you talking about?
Also, didn't know I had to split all these three up. Felt like that would be spamming.

**Comment by collinss** [comment]
No, it's much harder to keep track of individual bugs with more than one in the same issue. As for the issue template, I was referring to the system info, which at first glance looked like the generic info from the template. On closer inspection I realized that you included your system info at the end of the lines. However, it would be helpful if you could provide error logs, etc. "It doesn't work" doesn't really help us find the issue, much less fix it, so the more information you can provide, the better.

**Comment by MasterTextman** [comment]
I wish I could supply more information, but unfortunately I know next to nothing about Cinnamon. I don't know where error logs are located, and for every single one of these bugs it just "doesn't work".

When I say that, I moreso mean that nothing happens. I try to change the background, and it doesn't pop up an error or anything, it just doesn't change the background. When I try to set the shortcut to mute the volume, again, nothing happens.

If there are error logs I can share, please direct me to them because I am lost...

**Comment by collinss** [comment]
`~/.xsession-errors` and `~/.cinnamon/glass.log` are a good place to start. Also, if the error is with the settings window (such as your power settings bug), you can run `cinnamon-settings` from terminal - any errors that occur during use will be printed to the terminal window.

**Comment by MasterTextman** [comment]
```
Loading Power module
Traceback (most recent call last):
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 553, in button_press
    self.side_view_nav(widget, None, category)
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 113, in side_view_nav
    self.go_to_sidepage(cat, filtered_path)
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 126, in go_to_sidepage
    sidePage.build()
  File "/usr/share/cinnamon/cinnamon-settings/bin/SettingsWidgets.py", line 123, in build
    self.module.on_module_selected()
  File "/usr/share/cinnamon/cinnamon-settings/modules/cs_power.py", line 115, in on_module_selected
    device_types = [x[UP_TYPE] for x in self.csd_power_proxy.GetDevices()]
  File "/usr/lib/python2.7/dist-packages/gi/overrides/Gio.py", line 171, in __call__
    None)
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.cinnamon.SettingsDaemon was not provided by any .service files (2)
```

This was the error outputted by the terminal when I clicked it.

```
error t=2017-12-02T22:26:45.590Z Cinnamon Software Rendering mode enabled
info t=2017-12-02T22:26:46.140Z Cinnamon.AppSystem.get_default() started in 549 ms
info t=2017-12-02T22:26:46.149Z loading user theme: /usr/share/themes/Arc-Flatabulous-Dark/cinnamon/cinnamon.css
info t=2017-12-02T22:26:46.182Z added icon directory: /usr/share/themes/Arc-Flatabulous-Dark/cinnamon
info t=2017-12-02T22:26:46.310Z PlacesManager: Updating devices
info t=2017-12-02T22:26:46.372Z loaded at Sat Dec 02 2017 23:26:46 GMT+0100 (CET)
info t=2017-12-02T22:26:47.513Z Loaded applet menu@cinnamon.org in 1138 ms
info t=2017-12-02T22:26:47.626Z Loaded applet show-desktop@cinnamon.org in 112 ms
info t=2017-12-02T22:26:47.735Z Role locked: panellauncher
info t=2017-12-02T22:26:47.735Z Loaded applet panel-launchers@cinnamon.org in 106 ms
info t=2017-12-02T22:26:47.787Z Loaded applet window-list@cinnamon.org in 50 ms
info t=2017-12-02T22:26:47.841Z Role locked: notifications
info t=2017-12-02T22:26:47.846Z Loaded applet notifications@cinnamon.org in 58 ms
info t=2017-12-02T22:26:47.906Z Loaded applet user@cinnamon.org in 59 ms
info t=2017-12-02T22:26:47.913Z Loaded applet removable-drives@cinnamon.org in 6 ms
info t=2017-12-02T22:26:47.930Z Loaded applet keyboard@cinnamon.org in 16 ms
info t=2017-12-02T22:26:48.042Z Loaded applet network@cinnamon.org in 110 ms
info t=2017-12-02T22:26:48.144Z Loaded applet sound@cinnamon.org in 101 ms
info t=2017-12-02T22:26:48.153Z Loaded applet systray@cinnamon.org in 8 ms
info t=2017-12-02T22:26:48.221Z Loaded applet calendar@cinnamon.org in 66 ms
info t=2017-12-02T22:26:48.228Z Loaded applet windows-quick-list@cinnamon.org in 6 ms
info t=2017-12-02T22:26:48.279Z Loaded applet power@cinnamon.org in 50 ms
info t=2017-12-02T22:26:48.279Z AppletManager.init() started in 1906 ms
info t=2017-12-02T22:26:48.284Z Cinnamon took 2820 ms to start
info t=2017-12-02T22:26:48.529Z GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.cinnamon.SettingsDaemon was not provided by any .service files
trace t=2017-12-02T22:26:48.529Z 
<----------------
_proxyInvoker/asyncCallback@resource:///org/gnome/gjs/modules/overrides/Gio.js:83
---------------->
info t=2017-12-02T22:26:49.913Z Hiding systray: network
info t=2017-12-02T22:26:49.915Z Adding systray: blueman (24x24px)
info t=2017-12-02T22:26:50.971Z Adding systray: discord (24x24px)
```
And these were the contents of .cinnamon/glass.log.

As for xsession-errors, I switched back to GNOME for the time being, so I don't think it'll help that much. Maybe I'll switch back and post said log.

**Comment by leigh123linux** [comment]
Having gnome installed blocks cinnamon-settings-daemon, the issue was fixed in cinnamon-3.4.x

https://github.com/linuxmint/cinnamon-session/commit/402355658665f5d544452e39902213c1d9027895

**Comment by leigh123linux** [comment]
This indicates the issue is caused by cinnamon-settings-daemon not starting/running.

```
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.cinnamon.SettingsDaemon was not provided by any .service files 
```

**Comment by MasterTextman** [comment]
Does this mean I have to uninstall GNOME completely?

I ran the command of the settings from GNOME, yes, but it wasn't working when I was using Cinnamon as well.

**Comment by MasterTextman** [comment]
I think I'll split the other two issues seperately too...

**Comment by leigh123linux** [comment]
@MasterTextman TBH there is little point in filing your issue as we aren't going to fix old releases, try updating to cinnamon-3.6.x.

> I think I'll split the other two issues seperately too...
